api/routes.py

```diff
@@ -84,49 +84,49 @@ def get(self, query_args):
 
 
 fields = Blueprint(
-    "fields", __name__, description="Retrieving information about annotations fields"
+    "fields", __name__, description="Get information about annotations fields"
 )
 
 
 @fields.route("/api/fields")
 class FieldList(MethodView):
     @fields.response(200, FieldSchema(many=True))
     def get(self):
-        "Return all annotations field data."
+        "List all annotations fields."
         return Field.query.all()
 
 
 @fields.route("/api/fields/<string:name>")
 class FieldInformation(MethodView):
     @fields.response(200, FieldSchema)
     def get(self, name):
-        "Return data about a specific annotations field."
+        "Get information about an annotations field."
         return Field.query.get_or_404(name)
 
 
-tasks = Blueprint("tasks", __name__, description="Fetching and updating Toolhunt tasks")
+tasks = Blueprint("tasks", __name__, description="Get incomplete tasks and submit data to Toolhub.")
 
 
 @tasks.route("/api/tasks")
 class TaskList(MethodView):
     @tasks.response(200, TaskSchema(many=True))
     def get(self):
-        "Return a bundle of 10 incomplete tasks."
+        "Get a bundle of ten incomplete tasks."
         return Task.query.filter(Task.user.is_(None)).limit(10)
 
 
 @tasks.route("/api/tasks/<string:task_id>")
 class TaskById(MethodView):
     @tasks.response(200, TaskSchema)
     def get(self, task_id):
-        "Return information about a specific task."
+        "Get information about a specific task."
         task = Task.query.get_or_404(task_id)
         return task
 
     @tasks.arguments(TaskCompleteSchema)
     @tasks.response(201)
     def put(self, task_data, task_id):
-        """Update a tool record on Toolhub."""
+        """Update a record on Toolhub and in our database."""
```

Comment:
I prefer the previous version – the fact that we update the database is an implementation detail on the backend. Imo, "tool" is more descriptive than "record" too.